Lombiq.Analyzers.PowerShell/Rules/Measure-AutomaticVariableAlias/Measure-AutomaticVariableAlias.psm1

```diff
@@ -31,29 +31,29 @@ function Measure-AutomaticVariableAlias
         try
         {
             # Filter down tokens to just variable tokens with the name "_".
-            foreach ($automaticVariableAliasToken in $Token | Where-Object { $PSItem.GetType().Name -eq "VariableToken" -and $PSItem.Name -eq "_" })
+            foreach ($automaticVariableAliasToken in $Token | Where-Object { $PSItem.GetType().Name -eq 'VariableToken' -and $PSItem.Name -eq '_' })
             {
-                $correctionTypeName = "Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic.CorrectionExtent"
+                $correctionTypeName = 'Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic.CorrectionExtent'
                 $correctionExtent = New-Object -TypeName $correctionTypeName -ArgumentList @(
                     $automaticVariableAliasToken.Extent.StartLineNumber
                     $automaticVariableAliasToken.Extent.EndLineNumber
                     $automaticVariableAliasToken.Extent.StartColumnNumber
                     $automaticVariableAliasToken.Extent.EndColumnNumber
                     '$PSItem'
-                    'Replaced the usage of the alias of the automatic variable ''$_'' with its full name ''$PSItem''.'
+                    'Replaced the usage of the alias of the automatic variable "$_" with its full name "$PSItem".'
                 )
 
                 $suggestedCorrections = New-Object System.Collections.ObjectModel.Collection[$correctionTypeName]
                 $suggestedCorrections.add($correctionExtent) | Out-Null
 
                 $results += [Microsoft.Windows.Powershell.ScriptAnalyzer.Generic.DiagnosticRecord]@{
-                    "Extent" = $automaticVariableAliasToken.Extent
-                    "Message" = '''$_'' is an alias of the automatic variable ''$PSItem''. Please consider using the full name of this' +
-                    ' variable for consistency.'
-                    "RuleName" = "PSAvoidUsingAutomaticVariableAlias"
-                    "RuleSuppressionID" = "PSAvoidUsingAutomaticVariableAlias"
-                    "Severity" = "Warning"
-                    "SuggestedCorrections" = $suggestedCorrections
+                    'Extent' = $automaticVariableAliasToken.Extent
+                    'Message' = '"$_" is an alias of the automatic variable "$PSItem". Please consider using the full' +
+                    ' name of this variable for consistency.'
```

Comment:
How about this then:
```suggestion
                    'Message' = (
                        '"$_" is an alias of the automatic variable "$PSItem". Please consider using the full' +
                        ' name of this variable for consistency.')
```
... of course with rewrapped text?

Reply:
Actually, this can use the array join format, so we don't need to concat (I'd prefer that, because it's easier to rewrap), I'll commit that.

Reply:
I thought we agreed on this format in that other conversation?

Reply:
It's viable, but the join method is better, because it's easier to manage. It would be better to use the same syntax for both situations though, but the join method is superior. I'll reply in the other thread, let's continue there.